wire-ios/Wire-iOS/Sources/UserInterface/Settings/Backup/BackupActionCell.swift

```diff
@@ -0,0 +1,54 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+
+final class BackupActionCell: UITableViewCell {
+    let actionTitleLabel: DynamicFontLabel = {
+        let text = L10n.Localizable.Self.Settings.HistoryBackup.action
+        let label = DynamicFontLabel(text: text,
+                                     fontSpec: .normalRegularFont,
+                                     color: SemanticColors.Label.textDefault)
```

Comment:
suggestion: Since you're already on it, can you try using the new initializer and make sure it looks good with a bigger font size?

Reply:
Thanks for this suggestion, since this is not a main task I would like to keep the effort low and won't do it in this pull request.